### ISSUE TITLE
Temporary Storage toggle in Get Started Page does not respect preferred ephemeral type

### ISSUE THREAD

**Issue by sleshchenko** [eclipse-che/che#17725]
### Describe the bug
Storage toggle in Get Started Page does not respect preferred ephemeral type
<img width="707" alt="image" src="https://user-images.githubusercontent.com/17478559/91333602-69685f00-e7eb-11ea-81ea-1b71789a89b0.png">
### Che version

 - [ ] latest
 - [ ] nightly
 - [ ] other: 7.17.2

### Steps to reproduce
1. Deploy Che with configured
```
CHE_WORKSPACE_STORAGE_AVAILABLE__TYPES: ephemeral
CHE_WORKSPACE_STORAGE_PREFERRED__TYPE: ephemeral
```
2. Check that on Get Started page temporary storage toggle is disabled (persistent).

### Expected behavior
Temporary storage toggle is enabled as configured.

### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [ ] Openshift (include output of `oc version`)
   - [ ] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [ ] chectl
     * provide a full command that was used to deploy Eclipse Che (including the output)
     * provide an output of `chectl version` command
   - [ ] OperatorHub
   - [ ] I don't know

### Environment
   - [ ] my computer
       - [ ] Windows
       - [ ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [ ] other: please specify

**Comment by sleshchenko** [comment]
Setting P1 since it's pretty important for Hosted Che and ephemeral storage type should be the default.

**Comment by sleshchenko** [comment]
@ibuziuk not sure how critical it is for hosted che, so I just let you know about that. That bug seems to appear in 7.18.0 and is going to be fixed in 7.19.0
If it's needed to be backported - let us know.

**Comment by ibuziuk** [comment]
@sleshchenko thanks for heads up, I believe we would need it in 7.18.x since Hosted Che is supposed to run against the version that will be used in the next CRW (7.18.x for CRW 2.4)

**Comment by ibuziuk** [comment]
@sleshchenko @akurinnoy looks like the issue is still there for `7.18.1`, is it expected?

**Comment by akurinnoy** [comment]
@ibuziuk it supposed to be fixed. Is `7.18.1` based on `7.18.x`?

**Comment by ibuziuk** [comment]
yup, I think so

**Comment by ibuziuk** [comment]
![image](https://user-images.githubusercontent.com/1461122/92371534-02f11400-f0fc-11ea-856c-4dd76bad5cb7.png)

@sleshchenko @akurinnoy folks, I'm reopening - hopefully, it will be possible to figure out what went wrong and potentially provide a fix in `7.18.2` if needed

**Comment by sleshchenko** [comment]
@ibuziuk dashboard seems to work as Che Server tells it to do
![Screenshot_20200907_122952](https://user-images.githubusercontent.com/5887312/92372675-f671b900-f105-11ea-827c-a4b3701e627b.png)

**Comment by ibuziuk** [comment]
@sleshchenko hmm.. you are looking at 7.18.0 on production (so, the issue is expected to be fixed even in 7.18.0)?

**Comment by sleshchenko** [comment]
it's not expected to be fixed in 7.18.0 but as we see - after fix - persistent is going to appear on the rhche by default.

**Comment by ibuziuk** [comment]
ok, got it so the combination of https://github.com/redhat-developer/rh-che/pull/1981 + `7.18.1` rollout is expected to work on Hosted Che

**Comment by ibuziuk** [comment]
Thanks, folks
I can confirm that the issue is fixed in 7.18.1 when the correct configuration is used on the Hosted Che end - https://github.com/redhat-developer/rh-che/pull/1982

@vparfonov I believe would be really nice to remove the obsolete property from upstream - https://github.com/eclipse/che/blob/7.16.x/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L527-L533 (not sure if this is done already)
